vlm/src/main/scala/geotrellis/contrib/vlm/geotiff/GeoTiffReprojectRasterSource.scala

```diff
@@ -30,12 +30,13 @@ case class GeoTiffReprojectRasterSource(
   crs: CRS,
   reprojectOptions: Reproject.Options = Reproject.Options.DEFAULT,
   strategy: OverviewStrategy = AutoHigherResolution,
-  private[vlm] val targetCellType: Option[TargetCellType] = None
+  private[vlm] val targetCellType: Option[TargetCellType] = None,
+  val baseTiff: Option[MultibandGeoTiff] = None
```

Comment:
Why is it `private val` in `GeoTiffRasterSource` and here it is `public`? What is the difference?

Reply:
Oh, that's not right. That should be `private`.